libraries/SrcWrapper/src/stm32/stm32_eeprom.c

```diff
@@ -130,7 +132,7 @@ static inline uint32_t get_flash_end(void)
  * in order to prevent overwritting
  * program data
  */
-#if defined(STM32L0xx)
+#if defined(DATA_EEPROM_BASE)
```

Comment:
In that case this could be simply removed.
As if EEPROM exists then this define will be never used.

Reply:
section removed